core/deployment/src/main/java/io/quarkus/deployment/index/IndexDependencyConfig.java

```diff
@@ -12,18 +12,18 @@
      * The maven groupId of the artifact to index
      */
     @ConfigItem
-    String groupId;
+    public String groupId;
```

Comment:
Could this be made an immutable class, btw? Not necessarily in this PR.

Reply:
It's a config group. I'm not sure it's possible to make config groups immutable?